src/autoloads/quests.gd

```diff
@@ -10,6 +12,7 @@ func remove_quest(remove: PlayerQuest) -> void:
     for quest in quest_list:
         if quest.label == remove.label:
             quest_list.erase(quest);
+            completed_quests += 1;
```

Comment:
Do we need to consider a stopgap - eg. is there any way for total_quests to hit > 5 and cause issues in the game?
```suggestion
            completed_quests += 1;
            completed_quests = min(completed_quests, 5)
```

Reply:
fixed this up in newest commit!